recipes-extended/xen/files/defconfig

```diff
@@ -84,24 +114,26 @@ CONFIG_HAS_PCI=y
 CONFIG_VIDEO=y
 CONFIG_VGA=y
 CONFIG_HAS_VPCI=y
+# end of Device Drivers
 
-#
-# Deprecated Functionality
-#
-# CONFIG_PV_LDT_PAGING is not set
-CONFIG_DEFCONFIG_LIST="arch/x86/configs/x86_64_defconfig"
+CONFIG_EXPERT=y
```

Comment:
I didn't see where EXPERT was a requirement in the Kconfig for argo.  Is it required due to some config dependencies or am I just missing where it says it outright?

Reply:
I think ARGO used to be behind EXPERT.  Now it is behind UNSUPPORTED.

Reply:
_EXPERT_ to _UNSUPPORTED_ eh? I can only imagine reading between those lines :joy: